File: HWAppFoundation/src/main/java/com/hw/hwdroid/foundation/app/HBaseRecyclerViewHolder.java
package com.hw.hwdroid.foundation.app;

import android.support.v7.widget.RecyclerView;
import android.view.View;

import butterknife.ButterKnife;

/**
 * Base RecyclerView Holder
 * <p>
 * Created by ChenJ on 2016/10/9.
 */
public class HBaseRecyclerViewHolder extends RecyclerView.ViewHolder {

    public HBaseRecyclerViewHolder(View itemView) {
        super(itemView);
        ButterKnife.bind(this, itemView);
    }

}

File: HWAppFoundation/src/main/java/com/hw/hwdroid/foundation/app/HBaseRecyclerViewHolder.kt
package com.hw.hwdroid.foundation.app

import android.support.v7.widget.RecyclerView
import android.view.View
import butterknife.ButterKnife

/**
 * Base RecyclerView Holder
 *
 *
 * Created by ChenJ on 2016/10/9.
 */
open class HBaseRecyclerViewHolder(itemView: View) : RecyclerView.ViewHolder(itemView) {

    init {
        ButterKnife.bind(this, itemView)
    }

}


